libs/confman/inc/public/tfc/confman/detail/config_dbus_client.hpp

```diff
@@ -30,11 +32,13 @@ struct config_property {
 
 namespace dbus {
 static constexpr std::string_view property_name{ "config" };
-}
+static constexpr std::string_view config_path{ "Config" };
```

Comment:
changed path of config, should we change the `property_name` as well to capital first letter? @SindriTh

Reply:
Sounds good to me, what does the DBUS naming standard say?